### PR TITLE
Enhance config configuratiguration recorder with status

### PR DESCRIPTION
**Pull request by spangenberg**
Enhance for https://docs.aws.amazon.com/securityhub/latest/userguide/securityhub-pci-controls.html#pcidss-config-1

### CODE REVIEW

File: client/mocks/builders_test.go [hunk 1]
```diff
@@ -224,6 +226,30 @@ func buildCloudwatchLogsFiltersMock(t *testing.T, ctrl *gomock.Controller) clien
 	}
 }
 
+func buildConfigConfigurationRecordersMock(t *testing.T, ctrl *gomock.Controller) client.Services {
```
Comment (roneli) [inline client/mocks/builders_test.go:229]: I would move this to a new file, this is legacy testing code and we should slowly remove from it and depercate.

**Review by roneli — CHANGES_REQUESTED** [review]
LGTM, would move the tests to a separate file, see comment in review.

File: resources/config_configuration_recorders.go [hunk 2]
```diff
@@ -83,16 +120,57 @@ func fetchConfigConfigurationRecorders(ctx context.Context, meta schema.ClientMe
...
+	}
+	for _, configurationRecorder := range resp.ConfigurationRecorders {
+		var configurationRecorderStatus types.ConfigurationRecorderStatus
+		for _, s := range status.ConfigurationRecordersStatus {
+			if s.Name == configurationRecorder.Name {
+				configurationRecorderStatus = s
+				break
+			}
+		}
+		res <- configurationRecorderWrapper{
```
Comment (bbernays) [inline resources/config_configuration_recorders.go:144]: Can we add an `exists` type field? That way we can have positive confirmation of the record in the database compared to having to handle if the record is missing for whatever reason (not in config, permissions issue, etc)

**Review by roneli — APPROVED** [review]
LGTM, can merge after fixing @bbernays suggestion

**Pull request merged**